CHANGELOG.md

```diff
@@ -7,19 +7,51 @@ next release (yyyy-mm-dd)
 
 #### ⛔ Breaking Changes
 
+#### New Features
+
+#### Bug fixes, Minor Improvements
+
+### UI Changes
+
+1.47.0 (2023-07-05)
+-------------------
+### Backend Changes
+
+#### ⛔ Breaking Changes
+
 * [SPM] Due to a breaking change in OpenTelemetry's prometheus exporter ([details](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.80.0))
   metric names will no longer be normalized by default, meaning that the expected metric names would be `calls` and
   `duration_*`. Backwards compatibility with older OpenTelemetry Collector versions can be achieved through the following flags:
   * `prometheus.query.normalize-calls`: If true, normalizes the "calls" metric name. e.g. "calls_total".
   * `prometheus.query.normalize-duration`: If true, normalizes the "duration" metric name to include the duration units. e.g. "duration_milliseconds_bucket".
 
-
 #### New Features
```

Comment:
I would not consider any of these as new features, rather minor improvements.

Reply:
Okay, when I look at a feature this is something the user can take advantage of directly, so for me at least the tracegen one is a new feature. I will revise as you wish.

Reply:
agree about tracegen, but the hotrod changes are mostly "internal". So it `Configuration.Close()` next item, not something that would affect a regular user.